echomiddleware/behavior_logger.go

```diff
@@ -42,7 +42,7 @@ func BehaviorLogger(serviceName string, config KafkaConfig) echo.MiddlewareFunc
 
 			if err = next(c); err != nil {
 				c.Error(err)
-				behaviorLogger.Err = err.Error()
+				behaviorLogger.Err = err
```

Comment:
`behaviorLogger.Err` is string type. So `err` should be converted to string type.
I think original code is right.

Reply:
you are right，
I've re-downloaded the code and it's fine now